Makefile.in

```diff
@@ -155,7 +87,7 @@ docker-build: docker-build-dev
 	portworx/px-fuse:dev make
 
 px_version.c:
-	echo "const char *gitversion = \"$(shell git name-rev --name-only HEAD):$(shell git rev-parse HEAD)\";" > $@
+	echo "const char *gitversion = \"$(shell git rev-parse HEAD)\";" > $@
```

Comment:
can we not keep the branch name as well, along with commit id, why this needs to be reverted.